### PR TITLE
[1LP][RFR] Automate collections remediate

### PR DESCRIPTION
**Pull request by psav**
Automate collections remediate

### CODE REVIEW

File: cfme/automate/service_dialogs.py [hunk 1]
```diff
@@ -73,17 +73,16 @@ def is_displayed(self):
         )
 
 
-class DialogCollection(Navigatable):
+class DialogCollection(NavigatableMixin):
```
Comment (izapolsk) [inline cfme/automate/service_dialogs.py]: please make sure, that this is fixed in another PR

Comment (psav) [reply cfme/automate/service_dialogs.py]: I is already in another pr #5396

**Comment by mshriver** [comment]
Moved to WIPTEST because of TypeErrors on the __init__ calls for mismatched args.

Most may be taken care of by the combination of this PR and #5396 but I'd like to see some justification for the failures.

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by mshriver** [comment]
PRT results in 16259 are good, failing tests aren't related.

**Pull request merged**